db/migrate/20221101194105_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, null: false, foreign_key: true
```

Comment:
Please add `author_id` as a foreign key instead of `user_id` as it's specified in the requirements.